srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/SettingsView.php

```diff
@@ -678,6 +684,13 @@ public function getPvpRewardsRender($amount, $isWinner, $bracket, $month, $year,
                 var r = confirm("You sure you want to continue?");
                 return r;

```

Comment:
you can even return confirm() directly

never use `var`, use only `let` or `const`, in this case, but even below I suggest to use const, but checking the code below `tooltipList` is not used, you can even don't declare that variable probably and you don't need to use .map() but .forEach()

Keep in mind that .map() return an array that "should be" assigned to a variable, while .forEach() doesn't.

Anyways, these are minor things, if you have no time, I am totally fine with this

Reply:
Done!